include/sound/sof/info.h

```diff
@@ -26,6 +26,7 @@ enum sof_ipc_ext_data {
 
 /* FW version - SOF_IPC_GLB_VERSION */
 struct sof_ipc_fw_version {
+	struct sof_ipc_hdr hdr;
```

Comment:
this is confusing. The comment hints at a command, which would mean that you really wanted sof_ipc_cmd_hdr?